exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -66,6 +68,15 @@
    */
   String getUserAgentInfo();
 
+  /**
+   * Returns transaction with current status. Or {@code Optional.empty()} if
```

Comment:
`Returns the information about the transaction; or {@code Optional.empty()} if ?`?